index.html

```diff
@@ -1,29 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link
-            href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap"
-            rel="stylesheet"
-        />
-        <link rel="stylesheet" href="./styles/style.css" />
-        <link rel="stylesheet" href="./styles/reset.css" />
-        <link rel="stylesheet" href="./styles/global.css" />
 
-        <title>Cool Web</title>
-    </head>
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="./styles/style.css" />
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/global.css" />
 
-    <body>
-        <header>
-            <h1>Cool Web Project</h1>
-        </header>
+    <title>Cool Web</title>
+</head>
 
-        <main></main>
+<body>
+    <header>
+        A little bit about Alex...
+    </header>
 
-        <footer>&copy; 2021 Aspiring Software Developer</footer>
+    <main>
+        <h1>Alex Orlet</h1>
+        <h2>Portland, Oregon</h2>
+        <h3>He/Him</h3>
+        here are a few of his favorite things :)
+        <ul>
+            <li>Flowers</li>
+            <li>Bread</li>
+            <li>Frogs</li>
+            <li>A <span style="font-weight: bolder; color:blueviolet;">GOOD</span> Book</li>
+        </ul>
+        <p>A little known fact about our dear boy Alex: he has dual citizenship but his
```

Comment:
Whaaaaaaat?!?!?!?! That's awesome! Where is your dual citizenship with? Def need to get to an Embassy! haha!

Reply:
Got it from my french mama! I might head to Chicago on my spring break to visit family and the embassy there :)